packages/components/src/organisms/EmptyState/EmptyState.tsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react'
+import type { PropsWithChildren, ReactNode } from 'react'
+
+type Variant = 'default' | 'rounded'
+
+export interface EmptyStateProps {
+  /**
+   * ID to find this component in testing tools (e.g.: cypress, testing library, and jest).
+   */
+  testId?: string
+  /**
+   * The EmptyState component's title.
+   */
+  title?: string
+  /**
+   * A React component that will be rendered as an icon on the Title.
+   */
+  titleIcon?: ReactNode
+  /**
+   * Specifies the component border variant.
+   */
+  variant?: Variant
```

Comment:
I could not be able to identify the differences between the default and rounded variants, is this right?

![Screenshot 2023-04-17 at 10 44 46](https://user-images.githubusercontent.com/51174217/232502639-aa1f67fa-08da-4d77-85ab-0e2fa98817bd.png)

Reply:
There is a slight difference in the borders 👁️ 

<img width="273" alt="image" src="https://user-images.githubusercontent.com/3356699/232526356-206d9ad8-bfd2-48c0-a59e-d1e5a4c3f5df.png">